docs/developers/guard-accounts.md

```diff
@@ -0,0 +1,208 @@
+
+[comment]: # (mx-context-auto)
+
+### Change the Active Guardian
+
+Accounts can still be compromised by phishing attacks or other different ways. For unknown reasons, you may be confronted with the necessity to change the active guardian. The scope of account guarding is to protect users in case of a compromised account, but there are edge cases where it is uncertain whether a user or a scammer is changing the guardian, potentially leading to a scammer taking control of a user's funds. To address this issue, additional measures are necessary to allow users to protect their funds. One solution is the need for an activation time for setting a new guardian. If the transaction is not confirmed by an existing guardian, the activation time will give the user a chance to take action before the new guardian is set. In case the guardian confirms the transaction, the change will be made immediately. This solution will ensure that users have control over their accounts and can protect their funds even in the face of unexpected changes.
+
+Taking into account the aforementioned considerations for changing to a new guardian, it's worth noting that a user must send a new ```SetGuardian``` ([described here](/developers/built-in-functions#setguardian)) transaction to change guardians. This transaction may either be unguarded (meaning that the guardian does not have to co-sign it, so the transaction will not contain a guardian address on the ```guardian``` field, nor the ```guardianSignature```, with proper transaction version and option, see [transaction fields](/sdk-and-tools/rest-api/transactions#send-transaction)), or it may be a guarded transaction (in the event that the account was already guarded). In the case of an unguarded transaction, it will take 20 days to become active, whereas a guarded transaction will occur instantly.
+
+If the sent ```SetGuardian``` transaction is an unguarded one, the user will be able to see ```pendingGuardian``` set ([along with the activation epoch](/sdk-and-tools/rest-api/addresses#get-address-guardian-data)). On the other hand, if the transaction is a guarded one, the new guardian will become directly active.
+
+[comment]: # (mx-context-auto)
+
+## For users
+
+[comment]: # (mx-context-auto)
+
+### Wallet Extension
+
+The process of utilizing the wallet extension generally remains unchanged, with one additional step. Now, whenever you wish to initiate a guarded transaction, you will need to input the 2FA code for the TCS service. This step ensures that your transaction is properly signed by the guardian. Refer to [wallet-extension](/wallet/wallet-extension#guardian) page for more information.
+
+[comment]: # (mx-context-auto)
+
+### Web-Wallet
+
+Follow on details about web-wallet [here](/wallet/web-wallet.md#guardian).
+
+[comment]: # (mx-context-auto)
+
+### xPortal
+
+There is no solution released for the **xPortal Wallet**. As soon as there is one, it will be documented and referenced here.
```

Comment:
The solution for xPortal is still under implementation & testing and soon it will be documented and referenced here.. or something along these lines

Reply:
done